tests/builder/help.rs

```diff
@@ -964,6 +999,71 @@ OPTIONS:
     ));
 }
 
+#[test]
+#[cfg(all(feature = "wrap_help"))]
+fn possible_value_wrapped_help() {
+    #[cfg(feature = "unstable-v4")]
+    static WRAPPED_HELP: &str = "test 
+
+USAGE:
+    test [OPTIONS]
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+        --possible-values <possible_values>
+            Possible values:
+                - name: Long enough help message to clearly
+                          warrant wrapping
+                - second
+
+        --possible-values-with-new-line <possible_values_with_new_line>
+            Possible values:
+                - long enough name to trigger new line:
+                    Long enough help message to clearly warrant
```

Comment:
I know you are using tab-widths for indentation but I wonder if we should only indent by `"- "` so `long`` and `Long` are aligned

Reply:
Good question, not sure my self, I wanted to indent them to differentiate them from the possible value name.

Also, one thing I forgot to mention, how should spaces be handled? Generally I'd say having spaces in possible values is a bad CLI interface, but they can be there, and we need to decide how to display them. In the inline listing, we use `" {}"` whenever they contain spaces.

Reply:
> Good question, not sure my self, I wanted to indent them to differentiate them from the possible value name.

I had thought about that but we do have color and `:` to help differentiate

> Also, one thing I forgot to mention, how should spaces be handled? Generally I'd say having spaces in possible values is a bad CLI interface, but they can be there, and we need to decide how to display them. In the inline listing, we use " {}" whenever they contain spaces.

I was wondering about that.  Let's quote for consistency